sqlcollaborative/dbatools
# Fixes #1146 - Duplicate role creation in Export-SqlUser

Fixes # 

Changes proposed in this pull request:
 - Create a list and track the roles created at export time, skipping it if it already exists.

How to test this code: 
- Run Export-SqlLogin on a box with a multiple users assigned to the same custom role.

Has been tested on minimum requirements:
- This change should work on all versions Export-SqlLogin works on  

Has been tested on maximum requirements:
- This change should work on all versions Export-SqlLogin works on 

Tests for tester:
- No additional tests have been added.
